### PR TITLE
Fix cove template indentation

### PR DESCRIPTION
**Pull request by michaelwood**

### CODE REVIEW

**Comment by michaelwood** [comment]
Thanks @BibianaC  , we have a new way to write commit messages 
https://opendataservices.gitbook.io/developer-docs/code/code-python/code-git#commit-message-format other than that it looks great. I'm happy to fix the commit messages if you want.

**Comment by robredpath** [comment]
This is good, but lines 141-151 also have dodgy indentation (sorry!)

**Comment by BibianaC** [comment]
@robredpath I have fixed that block.

File: cove_360/templates/cove_360/explore.html [hunk 1]
```diff
@@ -226,23 +226,22 @@ <h3 class="panel-title panel-title-explore">
...
-          {% trans " meets the " %}
-        {% endif %}
-	
-	{% blocktrans %}the requirements of the <a href="http://www.threesixtygiving.org/standard/">360Giving Standard</a>. Making sure your data uses the standard is important. Otherwise it cannot be used by other people and applications working with 360Giving format data. {% endblocktrans %}
-</diV>
+        <div class="explore-help">{% trans "This means that the data" %}
+          {% if validation_errors %}
+            {% trans " does not meet " %}
+          {% else %}
+            {% trans " meets the " %}
```
Comment (robredpath) [inline cove_360/templates/cove_360/explore.html:240]: @BibianaC I just realised that this text is actually wrong - it should just be " meets " (as 'the' is already in the segement after). Sorry! Could I ask you to fix that as well?

Comment (BibianaC) [reply cove_360/templates/cove_360/explore.html]: I have deleted it.

**Pull request merged**